.github/workflows/publish-snapshot.yml

```diff
@@ -19,9 +19,10 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Set up Java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          distribution: 'zulu'
+          java-version: 8
```

Comment:
untested change, maybe we should add [on: workflow_dispatch](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow)?

Reply:
I don't think it's necessary given the low traffic on this repo. Should this fail, we can follow up on it 👍